src/Symfony/Component/Notifier/Bridge/Firebase/Tests/FirebaseTransportFactoryTest.php

```diff
@@ -11,50 +11,39 @@
 
 namespace Symfony\Component\Notifier\Bridge\Firebase\Tests;
 
-use PHPUnit\Framework\TestCase;
 use Symfony\Component\Notifier\Bridge\Firebase\FirebaseTransportFactory;
-use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
-use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Tests\TransportFactoryTestCase;
```

Comment:
This will break as soon as composer installs Notifier from the dist package because we exclude the Tests folder there.

Reply:
Okay, I've just read up the discussion. Forget what I've said.

Reply:
Ok and what shall I do?

We decided to introduce them against `5.1` in `Tests/` folder for ease maintenance and move them to `Test/` folder in `5.x` as experimental.

Reply:
Ok 😄